versioning-policy.md

```diff
@@ -64,3 +64,9 @@ release. Major releases do not happen according to a fixed schedule.
 | Mid Aug 2018 || Code freeze. Release branch cut.|
 | Late Aug 2018 || QA period. Focus on bug fixes, tests, stability and docs. Generally, no new features merged.|
 | Early Sep 2018 || Release candidates (RC), voting, etc. until final release passes|
+
+<h2>Maintenance Releases and EOL</h2>
+
+Feature release branches will, generally, be maintained with bug fixes releases for a period of 18 months. 
+For example, branch 2.1.x is no longer considered maintained as of July 2018, 18 months after the release 
+of 2.1.0 in December 2016. No more 2.1.x releases should be expected after that point, even for bug fixes.
```

Comment:
I'm fine with the 18 months, we should make sure everyone agrees. This essentially means we will be keeping 3 releases active.  And at this point 2.1.x will be dead. Did this go to the dev list?  I apologize if I missed it seems my yahoo mail account hasn't been getting everything for some reason.